pkg/controller/migstorage/validation.go

```diff
@@ -73,18 +71,6 @@ func (r ReconcileMigStorage) validate(storage *migapi.MigStorage) error {
 		return err
 	}
 
-	// Ready
-	storage.Status.SetReady(
-		!storage.Status.HasBlockerCondition(),
-		ReadyMessage)
-
-	// Apply changes.
-	storage.Status.EndStagingConditions()
```

Comment:
So validate (which runs near the beginning of reconcile) used to begin and end staging conditions, now it only begins staging. Makes sense.